### ISSUE TITLE
mat-checkbox checkmark SVG issue when using Ivy

### ISSUE THREAD

**Issue by mrnateriver** [angular/material2#15727]
Checkmark is invisible inside checkbox when using Ivy.
To be precise, SVG element has 0x0 size, doesn't have styles from its class for some reason and Chrome/Firefox don't even allow setting them manually for the element (this is a really peculiar issue).

[Screenshot 1](https://raw.githubusercontent.com/mrnateriver/angular-material-checkbox-issue/master/angular_material_checkbox_issue1.png)
[Screenshot 2: Chrome DevTools](https://raw.githubusercontent.com/mrnateriver/angular-material-checkbox-issue/master/angular_material_checkbox_issue2.png)

#### Bug, feature request, or proposal:
Bug.

#### What is the expected behavior?
Checkmark should be shown.

#### What is the current behavior?
It isn't.

#### What are the steps to reproduce?
https://github.com/mrnateriver/angular-material-checkbox-issue
1) `yarn install` or `npm install`;
2) Open `node_modules\@angular\common\fesm5\http.js` and comment out the last line (`export {HttpClientXsrfModule};`) - this fixes [a problem](https://github.com/angular/angular/issues/28068) in `ivy-ngcc` in Angular 7.x;
3) Run `ng serve`.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Chrome 73.0.3683.86 and Firefox 66.0.2.
@angular/core 7.2.12
@angular/platform-browser 7.2.12
@angular/material 7.3.7

**Comment by crisbeto** [comment]
This didn't come up during our manual testing of Material with Ivy. Not quite sure what might be going on. cc @kara

**Comment by crisbeto** [comment]
I just tried it out again and I wasn't able to reproduce it. @mrnateriver your issue might be because you're using an older version of Angular (`~7.2.0`). Have you tried with the latest 8.0 beta?

**Comment by mrnateriver** [comment]
No, I haven't tried 8.0. I've enabled Ivy in v7 because I thought since it is shipped with it, and v7 is stable, then Ivy should be stable as well, albeit experimental.

I presume you haven't used the repo I've provided, so it may be indeed due to the Angular version.

**Comment by crisbeto** [comment]
No, I built the latest Angular from source, as well as Material from the `ivy-2019` branch which is somewhat recent. A lot of things are being fixed quickly on Ivy so it's possible that whatever is on the 7.x branch is very outdated at this point.

**Comment by mrnateriver** [comment]
Checked with 8.0.0-beta.11 and everything works as expected.
So I'll close the issue. Thank you for the heads-up.

However, there is still one thing bugging me - why didn't Chrome nor Firefox allow adding custom styles to the SVG element? It must have something to do with how Ivy adds that element to DOM. I've spent ages trying to find out with no luck, so I've even resorted to posting a StackOverflow [question](https://stackoverflow.com/questions/55529554/how-to-ban-setting-element-styles-manually-in-browser-dev-tools).
